src/parser.cpp

```diff
@@ -376,54 +365,21 @@ namespace ada::parser {
    */
   std::optional<uint64_t> parse_ipv4_number(std::string_view input) {
     // If input is the empty string, then return failure.
-    if (input.empty()) {
-      return std::nullopt;
+    if (input.empty()) { return std::nullopt; }
```

Comment:
@lemire These changes makes a difference, and we can merge them according to the impact. Can you open a pull request?

Reply:
Hold on. I am working on putting everything in one neat PR.